tools/cdn.sh

```diff
@@ -1,27 +1,45 @@
+  ADMIN_CDN_EXISTS=$(aws s3 ls s3://assets.us.auth0.com/extensions/$EXTENSION_NAME/assets/ | grep "admin.$1.min.css")
+
+  if [ ! -z "$BUNDLE_EXISTS" ]; then
+    echo "There is already a $EXTENSION_NAME-$1.js in the cdn. Skipping cdn publish…"
+  else
+    aws s3 cp build/bundle.js s3://assets.us.auth0.com/extensions/$EXTENSION_NAME/$EXTENSION_NAME-$1.js --region us-west-1 --acl public-read
+    echo "$EXTENSION_NAME-$1.js uploaded to the cdn"
+  fi
 
-BUNDLE_EXISTS=$(aws s3 ls s3://assets.us.auth0.com/extensions/$EXTENSION_NAME/ | grep "$EXTENSION_NAME-$CURRENT_VERSION.js")
-CDN_EXISTS=$(aws s3 ls s3://assets.us.auth0.com/extensions/$EXTENSION_NAME/assets/ | grep "link.$CURRENT_VERSION.min.css")
-ADMIN_CDN_EXISTS=$(aws s3 ls s3://assets.us.auth0.com/extensions/$EXTENSION_NAME/assets/ | grep "admin.$CURRENT_VERSION.min.css")
+  if [ ! -z "$CDN_EXISTS" ]; then
+    echo "There is already a link.$1.min.css in the cdn. Skipping cdn publish…"
+  else
+    aws s3 cp dist/assets/link.$1.min.css s3://assets.us.auth0.com/extensions/$EXTENSION_NAME/assets/link.$1.min.css --region us-west-1 --cache-control "max-age=86400" --acl public-read
+    echo "link.$1.min.css uploaded to the cdn"
+  fi
 
-if [ ! -z "$BUNDLE_EXISTS" ]; then
-  echo "There is already a $EXTENSION_NAME-$CURRENT_VERSION.js in the cdn. Skipping cdn publish…"
-else
-  aws s3 cp build/bundle.js s3://assets.us.auth0.com/extensions/$EXTENSION_NAME/$EXTENSION_NAME-$CURRENT_VERSION.js --region us-west-1 --acl public-read
-  echo "$EXTENSION_NAME-$CURRENT_VERSION.js uploaded to the cdn"
-fi
+  if [ ! -z "$ADMIN_CDN_EXISTS" ]; then
+    echo "There is already a admin.$1.min.css in the cdn. Skipping cdn publish…"
+  else
+    aws s3 cp dist/assets/admin.$1.min.css s3://assets.us.auth0.com/extensions/$EXTENSION_NAME/assets/admin.$1.min.css --region us-west-1 --cache-control "max-age=86400" --acl public-read
+    echo "admin.$1.min.css uploaded to the cdn"
+  fi
+}
 
-if [ ! -z "$CDN_EXISTS" ]; then
-  echo "There is already a link.$CURRENT_VERSION.min.css in the cdn. Skipping cdn publish…"
-else
-  aws s3 cp dist/assets/link.$CURRENT_VERSION.min.css s3://assets.us.auth0.com/extensions/$EXTENSION_NAME/assets/link.$CURRENT_VERSION.min.css --region us-west-1 --cache-control "max-age=86400" --acl public-read
-  echo "link.$CURRENT_VERSION.min.css uploaded to the cdn"
-fi
+deploy_bundle $CURRENT_VERSION
+echo "Finished deploying $CURRENT_VERSION.js to the cdn"
 
-if [ ! -z "$ADMIN_CDN_EXISTS" ]; then
-  echo "There is already a admin.$CURRENT_VERSION.min.css in the cdn. Skipping cdn publish…"
-else
-  aws s3 cp dist/assets/admin.$CURRENT_VERSION.min.css s3://assets.us.auth0.com/extensions/$EXTENSION_NAME/assets/admin.$CURRENT_VERSION.min.css --region us-west-1 --cache-control "max-age=86400" --acl public-read
-  echo "admin.$CURRENT_VERSION.min.css uploaded to the cdn"
-fi
+deploy_bundle $MAJORMINOR_VERSION
```

Comment:
Should be publish to a major version tag as well?

Reply:
I'm not opposed to it. I was just trying to keep this PR in scope of what was asked.